llama-index-core/llama_index/core/evaluation/retrieval/metrics.py

```diff
@@ -61,6 +67,43 @@ def compute(
         )
 
 
+class MRR(BaseRetrievalMetric):
+    """Mean Reciprocal Rank (MRR): Sums up the reciprocal rank score for each relevant retrieved document.
+    Then divides by the count of relevant documents.
+    """
+
```

Comment:
I think this will break the current way we invoke `MRR` to be computed. While I agree that RR and this MRR computation is more technically correct, I feel its a bit pedantic (and, I'm pretty pedantic myself!) and might not be worth the hassle of ensuring backwards compatibility and risking breaking changes.

Reply:
Yeah that makes sense. I completely understand. 

What do you think about having a net new metric called SQMRR (single query mean reciprocal rank)? --> To the boost flexibility, for devs, in your evaluation suite

Reply:
Would `SQMRR` ultimately compute the `MRR` tho? i.e., after we take an average of all `SQMRR`?

Reply:
Yeah, to a certain degree. It would be more granular version (since it scores based on multiple docs each time)

I could implement it as an optional implementation, using your parameter idea. Incase the user wants multiple docs scored each time

Let me know what you think